examples/cpp-dll/src-dll1/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "dll1"
```

Comment:
Also could use a better name, and the directory too should be called `src-tauri`
```suggestion
name = "tauri-app"
```

Reply:
I've used tauri_app just to follow the snake_case as that's what the build outputs